src/memote/support/consistency.py

```diff
@@ -247,11 +251,15 @@ def find_inconsistent_min_stoichiometry(model, atol=1e-13):
             # singleton set!
             inc_minimal.add((met,))
             continue
+        if n_computed >= max_mets_computed:
```

Comment:
Do you have a clear picture what is taking so long for large models? Is it because there are many unconserved metabolites or are the net stoichiometries large sets and many integer cuts (line 279) are applied?

I think this is also a candidate to be improved in a next PR. When we can report leakage modes, I guess, having a handful of them reported is enough to get started on improving the model. However, it would be best to try this out with an inconsistent model ourselves.

Reply:
It is hard to tell because both tests give an `Error` in the index. I tried the yeastGEM model (1904 uncoserved metabolites) with the cap, GLPK and without symengine and it was rather fast (the inconsistent minimal stoichiometry test is the one that takes most of the time among the three). I have removed the cap and will test it further. Do you have any idea why it gives the "Error"? The test since to retrieve a reasonable result so I am wondering if it has to do with the format.

Reply:
What do you mean exactly by

> both tests give an Error in the index

which index are you talking about?

Reply:
Oh, sorry, I mean the report (I was looking at the _index.html_).

Reply:
I see, I'll have to try out your branch.